packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -86,9 +99,28 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
     })();
   }, [data, dataProvider, userId]);
 
+  /* @conditional-compile-remove(raise-hand) */
+  const theme = useTheme();
+  /* @conditional-compile-remove(raise-hand) */
+  let activePersona = '';
+  if (props.allowActiveBorder) {
+    const callingPalette = (theme as unknown as CallingTheme).callingPalette;
```

Comment:
see comment below about passing in the raisedHandBorderColor, we can't assume here the callingPalette is accessible